src/wazuh_db/wdb_agent.c

```diff
@@ -52,25 +45,79 @@ static const char *global_db_queries[] = {
 
 int wdb_sock_agent = -1;
 
-/* Insert agent. It opens and closes the DB. Returns 0 on success or -1 on error. */
-int wdb_insert_agent(int id, const char *name, const char *ip, const char *register_ip, const char *key, const char *group, int keep_date) {
+static const char *global_db_accesses[] = {
```

Comment:
Only a small comment, may you consider the name `global_db_commands` ?

Reply:
DONE